plugin/storage/kafka/options.go

```diff
@@ -141,6 +145,16 @@ func (opt *Options) AddFlags(flagSet *flag.FlagSet) {
 		defaultCompressionLevel,
 		"(experimental) compression level to use on messages. gzip = 1-9 (default = 6), snappy = none, lz4 = 1-17 (default = 9), zstd = -131072 - 22 (default = 3)",
 	)
+	flagSet.Int(
+		configPrefix+suffixLingerMS,
+		defaultLingerMS,
+		"(experimental) Number of milliseconds to delay before sending records to Kafka. Higher value reduce request to Kafka but increase latency.",
```

Comment:
```suggestion
		"(experimental) Number of milliseconds to delay before sending records to Kafka. Higher value reduce request volume to Kafka but increase latency and the possibility of data loss in case of process restart.",
```

Reply:
We might also consider linking to the Kafka [documentation](https://docs.cloudera.com/documentation/kafka/latest/topics/kafka_performance.html#kafka_performance_tuning_producers_lead_time)

Reply:
I think you meant this anchor: https://docs.cloudera.com/documentation/kafka/latest/topics/kafka_performance.html#kafka_performance_tuning_producers

+1 to add the link to the help string, although it might look exceptionally ugly in the terminal. Maybe use a shortener?